tenders-api/update_v7.sql

```diff
@@ -0,0 +1,18 @@
+
+/*
+Tenders DB Update script: v6 (buyer_user_details table)
+*/
+
+-- Drop table
+
+-- DROP TABLE public.buyer_user_details;
+
+CREATE TABLE public.buyer_user_details (
+	user_id int4 NOT NULL,
+	user_password varchar NOT NULL,
+	created_by varchar NOT NULL,
```

Comment:
The created/updated cols should follow the same conventions as the other tables really - i.e.:
```
created_by      VARCHAR(2000) NOT NULL,
created_at      TIMESTAMP     NOT NULL,
updated_by      VARCHAR(2000) ,
updated_at      TIMESTAMP);
```

Reply:
Yes - organisation_mapping table has same columns